### PR TITLE
first pass at docs for directory upload

### PR DESCRIPTION
**Pull request by Uzay-G**
A script to upload directories to a data source, listed in the docs.

Let me know if this is the right structure/approach @spolu

### CODE REVIEW

**Review by spolu — COMMENTED** [review]
Yep that looks great.

Just need to fix the localhost URL and maybe use uppercase for these global variables.

Nice catch for `.md` can you add it to the data source upsert page filter as well?

You should add in the explanation that it uploads only .txt .md and .pdf

**Comment by Uzay-G** [comment]
Mime type for .md is text/plain so that's not a problem on the upsert side. Changed it to use constants.

File: docs/src/pages/data-sources-overview.mdx [hunk 1]
```diff
@@ -130,6 +130,48 @@ The following parameters are accepted when inserting a document:
...
+We also have a script you can use to upload a directory's contents to your data source. Copy the following code into `upload.py` and then fill in the values for your Dust API key, the workspace id you are in, and the data source you want to upload to. Then run `python <dir>`, where `dir` is the directory from which you want to upload the documents.
+
+```python
+#!/usr/bin/env python
+import requests
+import pathlib
+import pdftotext
+import sys
+
+KEY = "DUST API KEY"
```
Comment (spolu) [inline docs/src/pages/data-sources-overview.mdx:144]:
```
// The three following variables need to be set with your own Dust API key (can be found in the interface),
// Workspace ID (can be found in the workspace URL) and Data Source ID:
DUST_API_KEY=""
DUST_WORKSPACE_ID=""
DUST_DATA_SOURCE_ID=""

ENDPOINT= f"https://dust.tt/api/v1/w/{WORKSPACE_ID}/data_sources/{DS}/documents/"
```

**Review by spolu — DISMISSED** [review]
LGTM!

Suggested a few usability changes, feel free to merge as is or after addressing them :+1:

**Review by spolu — APPROVED** [review]
You could have merged direclty :+1:

**Pull request merged**